auto/joequiz.q

```diff
@@ -0,0 +1,17 @@
+\d .qwz
+
+/ config
+channel:"pub-quizzing"
+
+getQuiz:{
+ / Getting json object of joe website links
+  root:system "wget -q https://www.joe.co.uk/joe/the-joe-friday-pub-quiz -O -";
```

Comment:
can we skip the `system` & use `.Q.hg`? or `.req.g` if needed (reQ is a slightly more full featured HTTP client, loaded with kdbslack by default - it's in the `util/` directory)

Reply:
Yep, can use `.req.g` instead, but what's the best way to parse it afterwards to get the correct week?

Reply:
Not sure, haven't looked at the return - I'd assume something similar to what you've done already? Return should just be HTML string, same as with `wget` I think?